events/templates/events/event_list.html

```diff
@@ -0,0 +1,57 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block title %}Guild's LiveLobby{% endblock %}
+
+{% block content %}
+<body class="bg-secondary">
+    
+    <!--Header Card-->
+    <header class="container-fluid text-white bg-dark border-0 rounded-0 h-18 mh-18 px-1 pt-0 pb-2 sticky-top">
+        <!-- Guild LiveLobby Banner -->
+        <div class="row no-gutters justify-content-center mb-1">
+            <div class="col-1 mr-1 align-self-center">
+                <a href="http://guild.house">
+                    <img class="float-right" style="height: 40px; width: 120px;" src="{% static 'events/guild_white.png' %}" alt="Guild Logo">
+                </a>
+            </div>
+            <div class="col-1 ml-1">
+                <p class="text-left" style="font-size: 30px; transform: translateY(15%)">LiveLobby</p>
+            </div>
+        </div>
+    
+        <!-- Create Event Button -->
```

Comment:
The only trouble with putting the button below the header is that it will not stay with the header when scrolling down the page since it would sit at the top of the body (ironically, the header would hide the button when scrolling haha)

Instead, I added a header_extension block, which can be implemented if people need to add things like buttons or a subtitle, extra navigation etc., but is otherwise not on the page if not implemented so the header can remain plain.